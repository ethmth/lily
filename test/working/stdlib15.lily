
let a::int = [-74,48,12]
a.mergesort()

print(a)