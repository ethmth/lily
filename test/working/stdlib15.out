[-74, 12, 48]
